### DISCUSSION TITLE
Nested Layout

### DISCUSSION THREAD

**Discussion by hakimLyon** [remix-run/remix#4367] [Q&A] [answered]
I would like any routes after /* to have the same layout except /dashboard.
That's possible?

**Comment by sergiodxa** [comment, marked as answer]
You can do that with [pathless layout routes](https://remix.run/docs/en/v1/api/conventions#pathless-layout-routes).

```
routes/__layout/index.tsx
routes/__layout/another-route.tsx
routes/__layout.tsx
routes/dashboard.tsx
```

**Reply by hakimLyon** [reply]
Thanks 😎
